src/App.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div id="app">
+    <Dogs
+      v-bind:dogs="dogs"
+      v-bind:selected="selected"
+      v-bind:onSelect="handleSelect"/>
+    <DogDetail
+      v-bind:dog="selected"/>
+  </div>
+</template>
+
+<script>
+import Dogs from './components/Dogs';
+import DogDetail from './components/DogDetail';
+import dogs from './dog-data';
```

Comment:
```suggestion
import dogs from './dog-data';
```
Currently, the path to this file is incorrect. The file is actually at the root of your project and should be moved into the src folder to match this relative path.